index.html

```diff
@@ -41,6 +67,9 @@ <h2>Adventure Awaits</h2>
       </div>
      
     </section>
+
+
+    
     <section class="content-destination">
       <h2>Pick Your Destination</h2>
       <p>Expedition excursion design darn excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
```

Comment:
it seems like your elements within your content container are taking the full width of the page